manifests/t/TeamRadarr/Radarr/3.2.2.5080/TeamRadarr.Radarr.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: TeamRadarr.Radarr
+PackageVersion: 3.2.2.5080

```

Comment:
```suggestion
PackageVersion: 3.2.2
```
This also shows as 3.2.2 🤦‍♂️
![image](https://user-images.githubusercontent.com/15158490/122393086-cc8ed200-cf74-11eb-870f-42973f56b338.png)